alert-common/src/main/java/com/synopsys/integration/alert/common/rest/api/ConfigurationCRUDHelper.java

```diff
@@ -48,7 +48,7 @@ public ConfigurationCRUDHelper(AuthorizationManager authorizationManager) {
         return new ActionResponse<>(HttpStatus.OK, optionalResponse.get());
     }
 
-    public <T> ActionResponse<T> create(Supplier<ValidationResponseModel> validator, Callable<T> createdModelSupplier, ConfigContextEnum context, DescriptorKey descriptorKey) {
+    public <T> ActionResponse<T> create(Supplier<ValidationResponseModel> validator, Callable<T> createdModelSupplier) {
```

Comment:
I think using Callable here and on line 68 implies concurrency where there isn't any. If we want to have the createdModelSupplier able to throw exceptions, we should instead use our own ThrowingSupplier for clarity.

I also personally prefer modelCreator to createdModelSupplier, but that's splitting hairs =)